test/libraries/MultiSendCallOnly.spec.ts

```diff
@@ -145,5 +147,26 @@ describe("MultiSendCallOnly", async () => {
                 ),
             ).to.be.eq("0x" + "baddad".padEnd(64, "0"));
         });
+
+        it("can bubble up revert message on call", async () => {
+            const { delegatecaller, multiSend, mock } = await setupTests();
+            await mock.givenCalldataRevertWithMessage("0xbaddad", "Some random message");
+
+            const txs: MetaTransaction[] = [
+                {
+                    to: mock.address,
+                    value: 0,
+                    data: "0xbaddad",
+                    operation: 0,
+                },
+            ];
+            const { data } = buildMultiSendSafeTx(multiSend, txs, 0);
+
+            const { success, returndata } = await delegatecaller.callStatic.makeDelegatecal(multiSend.address, data);
+            expect(success).to.be.false;
+            expect(returndata).to.be.equal(
+                "0x08c379a000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000013536f6d652072616e646f6d206d65737361676500000000000000000000000000",
```

Comment:
Instead of inserting plain hex data, I'd encode the revert message to make the test easier to follow. Same in other test files.

Reply:
Done